versioneer.py

```diff
@@ -927,7 +927,7 @@ def get_versions():
     return {"version": "0+unknown", "full-revisionid": None,
             "dirty": None,
             "error": "unable to compute version"}
-'''
+'''  # noqa: W605
```

Comment:
Yeah, I hit this one in archive as well. I chose to ignore the whole file, as 'not our code' instead, since any changes we make in it will get stomped on if/when versioneer is upgraded. It's been more than a years since the last release, so perhaps that's not a huge issue. :-)